packages/webpack/src/runtime/runtime-namespace.ts

```diff
@@ -3,6 +3,7 @@ import { LavaMoatPluginOptions } from '../plugin'
 
 type RequiredProperty<T, K extends keyof T> = Omit<T, K> & Required<Pick<T, K>>
 export interface RuntimeNamespace {
+  scuttle: {scuttle: <T, U>(arg1: T, arg2: U) => {}}
```

Comment:
I agree, but this is kind of how the assembler currently works, its current convention for assembling files is `<file-path/file-name>.js : <file-name>`, and since the word `scuttle` stands for both the file name and the exported function, we end up with that...

```
{
  name: 'scuttle',
  shimRequire: 'lavamoat-core/src/scuttle.js',
},
```

We could do:

```
{
  name: 'scuttling',
  shimRequire: 'lavamoat-core/src/scuttle.js',
},
```

But it introduces `scuttling` vs `scuttle` which isn't consistent with the rest of the assembled directives which I find unfortunate.

Thoughts?

Reply:
The latter looks better, because it's easier to match which name comes from where.

Reply:
f4a3bee33